### PR TITLE
3 making the buttons work

### LINKED ISSUE #3: Making the buttons work

### PR DESCRIPTION
**Pull request by pedrosilva410**
buttons must print to input

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,42 +1,56 @@
...
-            <button id="three" class="numbers" type="button">3</button>
-            <button id="mn" class="operations" type="button">-</button>
-            <button id="zero" class="numbers" type="button">0</button>
-            <button id="dot" class="numbers" type="button">.</button>
-            <button id="eq" type="button">=</button>
-            <button id="pl" class="operations" type="button">+</button>
+
+        <input id="operation-input" type="text">
+
+        <button id="leftparenthesis" class="operations" type="button">(</button>
```
Comment (201flaviosilva) [inline index.html]: Use camelCase for `id`

File: index.html [hunk 2]
```diff
@@ -1,42 +1,56 @@
...
         <p>coded with love by Pedro &#9829</p>
     </footer>
+
 </body>
-</html>
\ No newline at end of file
+
+</html>
+
+<script src="index.js"></script>
```
Comment (201flaviosilva) [inline index.html]: This should be inside `html` tag

File: index.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
+let input = document.getElementById('operation-input');
```
Comment (201flaviosilva) [inline index.js]: Please use `const` if you will not change the value

File: index.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+let rightparenthesis = document.getElementById('rightparenthesis')
+let rest = document.getElementById('rest')
+let clear = document.getElementById('allclear')
+let division = document.getElementById('division')
+let multiplication = document.getElementById('multiplication')
+let minus = document.getElementById('minus')
+let equals = document.getElementById('equals')
+let plus = document.getElementById('plus')
+
+zero.addEventListener('click', ()=>updateOperation('0'));
```
Comment (201flaviosilva) [inline index.js]: You probably could just reduce the code for this:

`document.getElementById('zero').addEventListener('click', ()=>updateOperation('0'));`

File: index.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+rest.addEventListener('click', ()=>updateOperation('%'));
+division.addEventListener('click', ()=>updateOperation('/'));
+multiplication.addEventListener('click', ()=>updateOperation('x'));
+minus.addEventListener('click', ()=>updateOperation('-'));
+plus.addEventListener('click', ()=>updateOperation('+'));
+clear.addEventListener('click', ()=>allclear())
+
+
+function updateOperation(operand) {
+    input.value=input.value+operand;
```
Comment (201flaviosilva) [inline index.js]: Just to reduce code, and to be more simple:

`input.value += operand`

File: index.js [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+  }
+
+function allclear() {
+    input.value=''
```
Comment (201flaviosilva) [inline index.js]: You missed the `;` :)

File: styles.css [hunk 7]
```diff
@@ -85,16 +92,24 @@ main{
...
+    color: #363636;
+    cursor: pointer;
+}
+
+
+#leftparenthesis{
     grid-area: 2/1/2/1;
 }
-#rp{
+#rightparenthesis{
```
Comment (201flaviosilva) [inline styles.css]: Use camelCase for `id`

Comment (201flaviosilva) [reply styles.css]: But, it’s nice you have updated the names :)

**Review by 201flaviosilva — COMMENTED** [review]
Just need to do some fixed, after that, it's ok

File: index.html (hunk 2 above)
Comment (201flaviosilva) [reply index.html]: Something like this:

```html
  </body>
  <script src="index.js"></script>
</html>
```

File: index.js (hunk 4 above)
Comment (201flaviosilva) [reply index.js]: Just if you will not use any more the button

File: index.html (hunk 1 above)
Comment (pedrosilva410) [reply index.html]: yup my bad

File: index.html (hunk 2 above)
Comment (pedrosilva410) [reply index.html]: ok

File: index.js (hunk 3 above)
Comment (pedrosilva410) [reply index.js]: ok

File: index.js (hunk 4 above)
Comment (pedrosilva410) [reply index.js]: yy

File: index.js (hunk 5 above)
Comment (pedrosilva410) [reply index.js]: i wasnt sure anymore how to do that ty

File: styles.css (hunk 7 above)
Comment (pedrosilva410) [reply styles.css]: ty

**Comment by 201flaviosilva** [comment]
By the way, use this type of file structure:

- LICENSE.md
- index.html
- README.md
- src/
  - script.js
  - style.css

File: LICENSE.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
+The MIT License (MIT)
+
+
+Copyright (c) 2021
```
Comment (201flaviosilva) [inline LICENSE.md:4]: Wrong year 😂

File: src/index.js [hunk 9]
```diff
@@ -0,0 +1,30 @@
+let input = document.getElementById('operationInput');
```
Comment (201flaviosilva) [inline src/index.js]: The `input` variable it's a `const`, you are only changing the properties

File: src/styles.css [hunk 10]
```diff
@@ -77,6 +77,13 @@ main{
     border-radius: 4px;
 }
 
+.operations:hover{
```
Comment (201flaviosilva) [inline src/styles.css:81]: nice

Comment (201flaviosilva) [reply src/styles.css:81]: It's not a big thing, but probably you want to do a [transition](https://www.w3schools.com/css/css3_transitions.asp). For the animation to be more smooth :)

**Review by 201flaviosilva — APPROVED** [review]
Something could be better, but it’s ok to merge :)

And nice to fix the things

File: LICENSE.md (hunk 8 above)
Comment (pedrosilva410) [reply LICENSE.md:4]: copied from u lol

File: src/index.js (hunk 9 above)
Comment (pedrosilva410) [reply src/index.js]: aight

File: src/styles.css (hunk 10 above)
Comment (pedrosilva410) [reply src/styles.css:81]: yup

File: LICENSE.md (hunk 8 above)
Comment (201flaviosilva) [reply LICENSE.md:4]: 🥲😅

**Pull request merged**